tools/param-name-importer/param-name-importer.csproj

```diff
@@ -9,10 +9,16 @@
   <ItemGroup>
     <!-- Not sure why needed, but only System.IO.Compression.FileSystem.dll is
       included by default, and ZipArchive is type forwarded to System.IO.Compression -->
-    <Reference Include="System.IO.Compression" />
+    <Reference Include="System.IO.Compression" Condition="'$(TargetFramework)' == 'net472'" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Xml.SgmlReader" Version="1.8.14" />
+    <!-- This package erroneously contains /netcoreapp3.1/SgmlReader.exe and /netcoreapp3.1/SgmlReaderDll.dll.
+         We are going to use a package reference to download the nuget, and a regular reference to actually
+         reference the correct assembly. -->
+    <PackageReference Include="Microsoft.Xml.SgmlReader" Version="1.8.16" ExcludeAssets="Compile" GeneratePathProperty="true" />
+    <Reference Include="SgmlReader">
+      <HintPath>$(PkgMicrosoft_Xml_SgmlReader)\lib\netstandard2.0\SgmlReaderDll.dll</HintPath>
```

Comment:
Where does `$(PkgMicrosoft_Xml_SgmlReader)` come from?  Is this a "NuGet-ism"?  I don't see anything with this string in the *current* NuGet package that's used (the one prior to this PR).

Reply:
It comes from `GeneratePathProperty="true"`.  This tells MSBuild/NuGet to create a magic property containing the path where it put the package.  Without this we can't know where the package is, as the `packages` folder can be moved in `NuGet.config`.